### PR TITLE
remove Stimulus from bundle

### PR DESCRIPTION
**Pull request by adrienpoly**
relates to : #26 

This is a tentative solution to remove Stimulus from the bundle. 

It requires that Stimulus is already installed and window.Stimulus is available. 
This has been the default for a long time and this code 👇  is very common now a days so I think we should be fine

```js
import { Application } from "@hotwired/stimulus"

const application = Application.start()

// Configure Stimulus development experience
application.debug = false
window.Stimulus = application

export { application }
```

maybe we can throw an error message if it is not present?

For action cable we don't really have a default object globally available so not sure how to handle this one.

### CODE REVIEW

**Comment by jorgemanrubia** [comment]
Thanks @adrienpoly 🙏. 

I'd prefer a more generic approach here to make sure that no dependencies on the project collide with app dependencies, rather than providing a per-dependency-specific solution. Since this is local development, I find hotwire-spark.js containing everything handy, even if there is duplicated code downloaded. But we certainly don't want any kind of interference with the app's dependencies. My plan was to validate the problem and understand all this better to see how to fix. See  https://github.com/basecamp/hotwire-spark/issues/26

Any investigation or exploration you want to make to fix this for good would be welcomed 🙏.

**Comment by jorgemanrubia** [comment]
@adrienpoly hey I'll close although I may reopen if we can't find something better here. Thanks again 🙏.

File: rollup.config.js [hunk 1]
```diff
@@ -41,5 +41,6 @@ export default {
       ],
       exclude: "node_modules/**",
     }),
-  ]
+  ],
+  external: ["@hotwired/stimulus"],
```
Comment (jorgemanrubia) [inline rollup.config.js:45]: Why did you add this? Not needed with your changes, right?

**Review by jorgemanrubia — COMMENTED** [review]
@adrienpoly after thinking a bit more about this, I think this one is great. Would you mind rebasing? I'll merge

File: rollup.config.js (hunk 1 above)
Comment (adrienpoly) [reply rollup.config.js:45]: yeah you are right it shouldn' be needed.

**Comment by adrienpoly** [comment]
@jorgemanrubia it is rebased and I removed the rollup config

**Comment by jorgemanrubia** [comment]
Thanks @adrienpoly, and sorry for the back and forth 🙏.
